.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- You can remove this section if you have contributed before -->
+### First time contributor checklist
+<!-- mark with x between the brackets -->
+- [ ] I have read the [Style Guidelines](https://priv.ly/pages/develop#CodingStandards)
+
+### Contributor checklist
+<!-- mark with x between the brackets -->
+- [ ] My changes target the latest develop branch
+- [ ] My contribution is fully baked and is ready to be merged as is
+- [ ] I have run the tests on these platforms:
```

Comment:
This should probably be broken into selenium and unit tests. It would also be good to link to the selenium/unit test documentation in the development guide.